packages/js/product-editor/src/hooks/use-product-variations-helper.ts

```diff
@@ -61,6 +102,7 @@ export function useProductVariationsHelper() {
 				},
 				{
 					delete: true,
+					default_values: defaultVariationValues,
```

Comment:
I decided to pass the `default_values` into the request from the client, instead of adding logic in the back end whether we wanted to use the default price or not.
This to me felled like a better long term solution from the API stand point and also more flexible.
The only downside is that it requires an additional request on the client side to get the default price.

Reply:
I disagree with that since it is a business rule to manage it like that and all business rules should be managed server side. But I can see your point.